test/test_errors.py

```diff
@@ -0,0 +1,18 @@
+from pymongo.errors import (AutoReconnect,
+                            NotMasterError,
+                            OperationFailure)
+from test import (client_context,
+                  client_knobs,
+                  PyMongoTestCase,
+                  sanitize_cmd,
+                  unittest)
+
+class TestErrors(PyMongoTestCase):
+    def test_not_master_error(self):
```

Comment:
Can you also add a similar test for OperationFailure?

Reply:
Done.